src/functions/horizBrace.js

```diff
@@ -0,0 +1,127 @@
+// @flow
+import defineFunction from "../defineFunction";
+import buildCommon from "../buildCommon";
+import mathMLTree from "../mathMLTree";
+import stretchy from "../stretchy";
+import Style from "../Style";
+
+import * as html from "../buildHTML";
+import * as mml from "../buildMathML";
+
+defineFunction({
+    type: "horizBrace",
+    names: ["\\overbrace", "\\underbrace"],
+    props: {
+        numArgs: 1,
+    },
+    handler(context, args, optArgs) {
+        const base = args[0];
+        return {
+            type: "horizBrace",
+            label: context.funcName,
+            isOver: /^\\over/.test(context.funcName),
+            base: base,
+        };
+    },
+    htmlBuilder(group, options) {
+        const style = options.style;
+
+        const hasSupSub = (group.type === "supsub");
+        let supSubGroup;
+        let newOptions;
+        if (hasSupSub) {
+            // Ref: LaTeX source2e: }}}}\limits}
+            // i.e. LaTeX treats the brace similar to an op and passes it
+            // with \limits, so we need to assign supsub style.
+            if (group.value.sup) {
+                newOptions = options.havingStyle(style.sup());
+                supSubGroup = html.buildGroup(group.value.sup, newOptions, options);
+            } else {
+                newOptions = options.havingStyle(style.sub());
+                supSubGroup = html.buildGroup(group.value.sub, newOptions, options);
+            }
+            group = group.value.base;
+        }
+
+        // Build the base group
+        const body = html.buildGroup(
+        group.value.base, options.havingBaseStyle(Style.DISPLAY));
+
+        // Create the stretchy element
+        const braceBody = stretchy.svgSpan(group, options);
+
+        // Generate the vlist, with the appropriate kerns               ┏━━━━━━━━┓
+        // This first vlist contains the subject matter and the brace:   equation
+        let vlist;
+        if (group.value.isOver) {
+            vlist = buildCommon.makeVList({
+                positionType: "firstBaseline",
+                children: [
+                    {type: "elem", elem: body},
+                    {type: "kern", size: 0.1},
+                    {type: "elem", elem: braceBody},
+                ],
```

Comment:
I need to double check, but I don't think it was ending up on the same element as before the change, but it didn't seem to have an impact so I removed it.  @ronkok can you think of any cases where having this class would matter?

Reply:
@kevinbarabash I added that class when we changed from background-image SVGs to inline SVGs in PR #807. My experience then was  if that `text-align: left` was not applied on just the right span, then the elements would render smaller, lower, and off to the right. It was a large, very noticeable difference.

I don't know why it would be different now.